src/main/java/org/example/bookshop/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package org.example.bookshop.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.example.bookshop.dto.UserRegistrationRequestDto;
+import org.example.bookshop.dto.UserResponseDto;
+import org.example.bookshop.exception.RegistrationException;
+import org.example.bookshop.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/authorization")
```

Comment:
```suggestion
@RequestMapping("/auth")
```
Let's follow technical requirements